lib/TownCrier/Handler/API/Event.pm

```diff
@@ -13,12 +13,38 @@ sub list {
                                  { sort => sub { my ($a, $b) = @_; $b->ordered_compare($a) } })} ] }
 }
 
+sub list_all {
+    my $db = var 'db';
+    { events => [ map { $_->rest }
```

Comment:
This seems egregious, but it's already there, so 🤷‍♂️.

Reply:
What's the problem? That it uses a terrible pattern, or that it exists at all? I can accept either, I just put it there because everything else seemed to have similar shapes.

(also TIL "egregious" also once meant "remarkably good")